plugins/modules/openssh_cert.py

```diff
@@ -46,6 +47,12 @@
             - If the private key is on a PKCS#11 token (I(pkcs11_provider)), set this to the path to the public key instead.
             - Required if I(state) is C(present).
         type: path
+    passphrase:
```

Comment:
How about renaming this to `signing_key_passphrase`? This would be more similar to `privatekey_passphrase` in `openssl_csr`, `x509_certificate` etc.